### PR TITLE
New device tracker for TP-Link EAP

### PR DESCRIPTION
**Pull request by ghost**
## Proposed change

Documentation for a new device tracker for TP-Link EAP access point.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/41583
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by disforw — DISMISSED** [review]
Well written, follows formatting

File: source/_integrations/tplink_eap.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: TP-Link EAP
+description: Instructions on how to use a TP-Link EAP access point as a device tracker.
+logo: tp-link.jpg
```
Comment (frenck) [inline source/_integrations/tplink_eap.markdown]: This should not be used for new documentation, use our Brands repository instead (and remove this line).

File: source/_integrations/tplink_eap.markdown [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ha_release: 0.117
+ha_domain: tplink_eap
+---
+
+This platform allows you to detect presence by looking at devices connected to a [TP-Link EAP](https://www.tp-link.com/us/business-networking/ceiling-mount-access-point/).
+
+<div class='note warning'>
+This integration uses the web interface to retrieve data from the device. The device webserver does not support concurrent connection.
+If you try to login to the interface, your session will be terminated when the integration will poll the device status.
+<br>
```
Comment (frenck) [inline source/_integrations/tplink_eap.markdown:17]: Please use markdown as much as possible.

File: source/_integrations/tplink_eap.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+logo: tp-link.jpg
+ha_category:
+  - Presence Detection
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_release: 0.117
+ha_domain: tplink_eap
+---
+
+This platform allows you to detect presence by looking at devices connected to a [TP-Link EAP](https://www.tp-link.com/us/business-networking/ceiling-mount-access-point/).
```
Comment (frenck) [inline source/_integrations/tplink_eap.markdown]:
```suggestion
This integration allows you to detect presence by looking at devices connected to a [TP-Link EAP](https://www.tp-link.com/us/business-networking/ceiling-mount-access-point/).
```

File: source/_integrations/tplink_eap.markdown [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+<div class='note warning'>
+This integration uses the web interface to retrieve data from the device. The device webserver does not support concurrent connection.
+If you try to login to the interface, your session will be terminated when the integration will poll the device status.
```
Comment (frenck) [inline source/_integrations/tplink_eap.markdown]:
```suggestion
This integration uses the web interface to retrieve data from the device. The device web server does not support concurrent connections.
If you try to login into the interface, your session will be terminated when the integration will poll the device status.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @chemicalstorm!

I've reviewed it and left some comments. Could you please take a look? Thanks! 👍

File: source/_integrations/tplink_eap.markdown (hunk 1 above)
Comment (ghost) [reply source/_integrations/tplink_eap.markdown]: Line has been removed.

File: source/_integrations/tplink_eap.markdown (hunk 3 above)
Comment (ghost) [reply source/_integrations/tplink_eap.markdown]: Integrated.

File: source/_integrations/tplink_eap.markdown (hunk 4 above)
Comment (ghost) [reply source/_integrations/tplink_eap.markdown]: Integrated.

File: source/_integrations/tplink_eap.markdown (hunk 2 above)
Comment (ghost) [reply source/_integrations/tplink_eap.markdown:17]: Could I have some guidance for that piece?
I was thinking this fell under the valid use of HTML according to https://developers.home-assistant.io/docs/documenting/create-page#html

My rationale for using the HTML: I feel that this is somewhat of importance to users of that integration, so I would be a bit reluctant to not have that piece of information highlighted in some way.

**Comment by ghost** [comment]
Unfortunately I've not found the time to fully address all the comments, and I've gradually moved most of my device tracking to using the app instead. I'll be closing this PR then.

**Pull request closed without merging**